sdks/typescript/package.json

```diff
@@ -85,6 +85,7 @@
     "long": "^5.3.1",
     "nice-grpc": "^2.1.12",
     "nice-grpc-common": "^2.0.2",
+    "prom-client": "^15.1.3",
```

Comment:
would it be possible to make this dependency optional? it was a mistake to not do that in the Python SDK